packages/manager/apps/container/src/container/legacy/server-sidebar/universe/WebSidebar.tsx

```diff
@@ -151,7 +152,19 @@ export default function WebSidebar() {
         },
       });
     }
-
+    if (features['web-hosting:wordpress-managed']) {
+      menu.push({
+        id: 'web-hosting-websites',
+        label: t('sidebar_web_hosting_wordpress_managed'),
+        icon: getIcon('ovh-font ovh-font-domain'),
+        badge: 'alpha',
+        routeMatcher: new RegExp('^/managedCMS/availableCMS'),
```

Comment:
```suggestion
        routeMatcher: new RegExp('^/wordpress-managed'),
```
If your url is wordpress-managed, your route matcher would be wordpress-managed